src/openms/source/CHEMISTRY/EnzymaticDigestion.cpp

```diff
@@ -80,11 +80,11 @@ namespace OpenMS
 
   EnzymaticDigestion::Specificity EnzymaticDigestion::getSpecificityByName(const String& name)
   {
-    for (Size i = 0; i < SIZE_OF_SPECIFICITY; ++i)
+    for (Size i = 0; i < SPEC_UNKNOWN; ++i)
```

Comment:
SIZE_OF_SPECIFICITY is a safer pattern (until we switch to enum classes)

Reply:
Add it or use SIZE as unknown?

Reply:
I would add SIZE as last and have UNKNOWN extra. At least that's how others do it